### DISCUSSION TITLE
Randomization of qutebrowser fingerprint

### DISCUSSION THREAD

**Discussion by crocket** [qutebrowser/qutebrowser#7140] [Ideas] [answered]
Disabling canvas_reading and webgl doesn't randomize canvas fingerprint and webgl fingerprint.

Brave browser randomizes canvas fingerprint, webgl fingerprint, browser plugin details, audiocontext fingerprint, and hardware concurrency.

Can qutebrowser randomize canvas, webgl, audiocontext, and hardware concurrency?

**Comment by The-Compiler** [comment, marked as answer]
Looking at [how Brave implemented this](https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections#fingerprinting-methods-randomized), it's based on (many, and apparently Brave-specific) patches on top of their Chromium sources. So I'm afraid that's not going to happen unless someone ports that all to QtWebEngine (and someone there is willing to maintain all of it too). I'd say that's unlikely to happen, unfortunately.

**Reply by crocket** [reply]
Too bad.

**Reply by crocket** [reply]
Does qutebrowser or qtwebengine do any fingerprint randomization now?
